packages/sil-voting-app/src/routes/login/index.tsx

```diff
@@ -1,11 +1,27 @@
 import Block from 'medulas-react-components/lib/components/Block';
```

Comment:
For now, place the icon in: `src/routes/login/assets`, if you need to use the same icon once again we can move it back to the general folder.